src/utils/config_parser/index.ts

```diff
@@ -42,7 +46,7 @@ export const getOwnershipRules = async (
 ): Promise<OwnershipRules> => {
   const config = await context?.config("approveman.yml");
   context?.log.info(`Found config: ${JSON.stringify(config)}`);
-  if (config !== null) {
+  if (config !== null && config !== undefined) {
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6106171)